iam-login-service/src/main/java/it/infn/mw/iam/config/security/IamApiSecurityConfig.java

```diff
@@ -66,7 +66,7 @@ public static class IamProxyCertificateApiConfig extends WebSecurityConfigurerAd
     @Override
     protected void configure(final AuthenticationManagerBuilder auth) throws Exception {
       auth.userDetailsService(userDetailsService)
-        .passwordEncoder(NoOpPasswordEncoder.getInstance());
+        .passwordEncoder(new BCryptPasswordEncoder());
```

Comment:
questo è corretto, ma va in (o almeno _anche_) `iam/iam-login-service/src/main/java/it/infn/mw/iam/config/security/IamTokenEndointSecurityConfig.java`